doc/release-notes-16192.md

```diff
@@ -0,0 +1,12 @@
+Wallet: Transaction fees
+------------------------
+
+Bitcoin Core caps fees at `-maxtxfee=<x>` (default: 
```

Comment:
How about `Currently the wallet caps fees at ...`?

Reply:
Adjusted